code/src/java/pcgen/base/geom/GridPoint.java

```diff
@@ -121,8 +121,8 @@ public Number getPreciseY()
 	@Override
 	public void setLocation(double x, double y)
 	{
-		this.x = x;
-		this.y = y;
+		this.x = java.lang.Double.valueOf(x);
```

Comment:
Becuase it imports Point2D, which has Point2D.Double and thus there are two "Double"s that need disambiguation.